unified_planning/solvers/solver.py

```diff
@@ -93,7 +122,15 @@ def ground(self, problem: 'up.model.Problem') -> Tuple[Problem, Callable[[Plan],
 
         Also, the "up.solvers.grounder.lift_plan" function can be called, if retrieving the needed map
         fits the solver implementation better than retrieving a function.'''
-        raise NotImplementedError
+        raise
+
+    @staticmethod
+    def credits(stream: Optional[IO[str]] = sys.stdout, full_credits: bool = False):
+        '''
+        This method takes an IO[str] and writes the credits of the planner onto the stream.
+        '''
+        if stream is not None:
+            credits.write_credits(stream, full_credits)
```

Comment:
```suggestion
    @staticproperty
    def credits() -> Optional[Credits]:
        '''
        This method returns the credits for this solver, that will be printed when the solver is used.
        If this function returns None, it means no credits to print.
        '''
        return None
```

PS: you need to define static properties as follows:
```
class staticproperty(staticmethod):
    def __get__(self, *args):
        return self.__func__()
```